sentences/cs/_common.yaml

```diff
@@ -9,11 +9,11 @@ responses:
     handle_error: "Při zpracování došlo k neočekávané chybě."
 lists: {}
 expansion_rules:
-  name: "{name}"
-  area: "{area}"
-  turn_on: "zapni"
-  turn_off: "vypni"
-  turn_on_light: "rozsviť"
-  turn_off_light: "zhasni"
+  nazev: "{name}"
```

Comment:
@balloob did I get it right?

Reply:
Yep!

Reply:
well no. The ones that wrap a slot should not be translated.